src/local_scheduler/local_scheduler.cc

```diff
@@ -1005,6 +1007,8 @@ void process_message(event_loop *loop,
         TaskExecutionSpec(from_flatbuf(*message->execution_dependencies()),
                           (TaskSpec *) message->task_spec()->data(),
                           message->task_spec()->size());
+    /* Set the tasks's local scheduler entrypoint time. */
+    execution_spec.setLastTimeStamp(current_time_ms());
```

Comment:
How critical is including the timestamp (for performance reasons)?

Reply:
it's critical to timestamp the execution spec at the two entry points into the local scheduler. And it's critical that the timestamps are only done inside the local scheduler, to make sure the diff (that will be taken during spillback cleanup) is accurate.